test/unit/healthcheck.spec.ts

```diff
@@ -0,0 +1,58 @@
+// tslint:disable:no-conditional-assignment
+
+import container from './test.inversify.config';
+import * as chai from 'chai';
+import * as sinon from 'sinon';
+import { errorHandler, requestHandler } from '../../source/healthCheckHandler';
+
+const expect = chai.expect;
+
+describe('healthcheck', () => {
+    describe('requestHandler', () => {
+
+        const sandbox: sinon.SinonSandbox = sinon.createSandbox();
+
+        afterEach(() => {
+            sandbox.restore();
+        });
+
+        const testCases = [
+            { statusCode: 200, exitCallArg: 0 },
```

Comment:
Note that this code is executed as a separate node process by docker itself - it is fully expected that each invocation exits with some kind of exit code.